### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @sdroege @EPashkin

### CODE REVIEW

File: src/auto/pixbuf.rs [hunk 1]
```diff
@@ -256,11 +256,11 @@ impl Pixbuf {
     //    unsafe { TODO: call ffi::gdk_pixbuf_save_to_buffer() }
     //}
 
-    //pub fn save_to_callback<'a, P: Into<Option</*Unimplemented*/Fundamental: Pointer>>, Q: Into<Option<&'a Error>>>(&self, save_func: /*Unknown conversion*//*Unimplemented*/PixbufSaveFunc, user_data: P, type_: &str, error: Q, : /*Unknown conversion*//*Unimplemented*/Fundamental: VarArgs) -> bool {
+    //pub fn save_to_callback<'a, P: FnMut(&Vec<u8>, usize, &Error) -> bool, Q: Into<Option<&'a Error>>>(&self, save_func: P, type_: &str, error: Q, : /*Unknown conversion*//*Unimplemented*/Fundamental: VarArgs) -> bool {
     //    unsafe { TODO: call ffi::gdk_pixbuf_save_to_callback() }
     //}
 
-    //pub fn save_to_callbackv<P: Into<Option</*Unimplemented*/Fundamental: Pointer>>>(&self, save_func: /*Unknown conversion*//*Unimplemented*/PixbufSaveFunc, user_data: P, type_: &str, option_keys: &[&str], option_values: &[&str]) -> Result<(), Error> {
+    //pub fn save_to_callbackv<P: FnMut(&Vec<u8>, usize, &Error) -> bool>(&self, save_func: P, type_: &str, option_keys: &[&str], option_values: &[&str]) -> Result<(), Error> {
```
Comment (sdroege) [inline src/auto/pixbuf.rs]: Hm why can't this be generated?

**Comment by sdroege** [comment]
:+1:

File: src/auto/pixbuf.rs (hunk 1 above)
Comment (GuillaumeGomez) [reply src/auto/pixbuf.rs]: Array of pointers aren't handled for the moment. :3

Comment (sdroege) [reply src/auto/pixbuf.rs]: Ah but that's just an array of strings, we have special support for that ;) Another time!

**Pull request merged**